recipe/recipe.yaml

```diff
@@ -0,0 +1,576 @@
+        - ${{ stdlib('c') }}
+        - ${{ compiler('fortran') }}
+        - cmake
+        - {if: win, then: ninja, else: make}
+        - pkg-config
+      host:
+        - if: osx
+          then:
+            - llvm-openmp
+        - if: linux
+          then:
+            - libgomp
+        - libblas
+        - libcblas
+        - liblapack
+      ignore_run_exports:
+        from_package:
+          - libblas
+          - libcblas
+          - liblapack
+          - ${{ fortran_compiler }}
+      run_exports:
+        - ${{ pin_subpackage('libsuitesparseconfig') }}
+    tests:
+      - package_contents:
+          lib:
+            - suitesparseconfig
+  - package:
+      name: libamd
+      version: 3.3.3
+    about:
+      license: BSD-3-Clause
+      license_file: AMD/Doc/License.txt
+    build:
+      script:
+        file: ${{ 'build-one.sh' if not win else 'build-one.bat' }}
+        env:
+          SUBPKG_DIR: AMD
+    requirements:
+      build:
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
+        - ${{ compiler('fortran') }}
+        - cmake
+        - {if: win, then: ninja, else: make}
+        - pkg-config
+      host:
+        - ${{ pin_subpackage('libsuitesparseconfig', exact=True) }}
+      run_exports:
+        - ${{ pin_subpackage('libamd') }}
+    tests:
+      - package_contents:
+          lib:
+            - amd
+  - package:
+      name: libbtf
+      version: 2.3.2
+    about:
+      license: LGPL-2.1-or-later
+      license_file: BTF/Doc/License.txt
+    build:
+      script:
+        file: ${{ 'build-one.sh' if not win else 'build-one.bat' }}
+        env:
+          SUBPKG_DIR: BTF
+    requirements:
+      build:
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
+        - cmake
+        - {if: win, then: ninja, else: make}
+        - pkg-config
+      host:
+        - ${{ pin_subpackage('libsuitesparseconfig', exact=True) }}
+      run_exports:
+        - ${{ pin_subpackage('libbtf') }}
+    tests:
+      - package_contents:
+          lib:
+            - btf
+  - package:
+      name: libcamd
+      version: 3.3.3
+    about:
+      license: BSD-3-Clause
+      license_file: CAMD/Doc/License.txt
+    build:
+      script:
+        file: ${{ 'build-one.sh' if not win else 'build-one.bat' }}
+        env:
+          SUBPKG_DIR: CAMD
+    requirements:
+      build:
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
+        - cmake
+        - {if: win, then: ninja, else: make}
+        - pkg-config
+      host:
+        - ${{ pin_subpackage('libsuitesparseconfig', exact=True) }}
+      run_exports:
+        - ${{ pin_subpackage('libcamd') }}
+    tests:
+      - package_contents:
+          lib:
+            - camd
+  - package:
+      name: libccolamd
+      version: 3.3.4
+    about:
+      license: BSD-3-Clause
+      license_file: CCOLAMD/Doc/License.txt
+    build:
+      script:
+        file: ${{ 'build-one.sh' if not win else 'build-one.bat' }}
```

Comment:
I believe `build-one` would be enough, and rattler-build will add the appropriate `.sh` or `.bat` ending :)

Reply:
Awesome! I tried to figure this out in the docs, but couldn't find an example. An example of this in the multi-output docs would be fantastic.